eternalcore-paper/src/main/java/com/eternalcode/paper/PaperOverlay.java

```diff
@@ -0,0 +1,22 @@
+package com.eternalcode.paper;
+
+import org.bukkit.entity.Player;
+
+import java.util.function.Consumer;
+
+public enum PaperOverlay {
+
+    ELDER_GUARDIAN("Elder Guardian", player -> player.showElderGuardian(false)),
```

Comment:
we can change it for one field using BiConsumer<Player, Boolean>

Reply:
So in order to change that, I have to change the constructor for this class, and effectively - PaperFeature too?